### PR TITLE
Changed the question to one with no ambiguity

### LINKED ISSUE #40410: "Wrong answer" error in Jupyter cells lesson of Data Analysis with Python course
The correct answer for the question: Which cells are responsible for rich display? is Markdown Cells.
But it erroneously accepts "Code Cells" as the correct answer.

To reproduce the bug, go to below link and select the correct answer to the question.

[link](https://www.freecodecamp.org/learn/data-analysis-with-python/data-analysis-with-python-course/jupyter-notebooks-cells)

Screenshot 2020-12-09 164523

### PR DESCRIPTION
**Pull request by yaseen221**
Checklist:

- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

Closes #40410

### CODE REVIEW

**Comment by yaseen221** [comment]
This change has been discussed upon on issue #40410. This change seems the most appropriate approach to resolve the issue.

**Review by ShaunSHamilton — DISMISSED** [review]
This LGTM 👍 

---

For future reference, once this is merged, your `master` branch will be spoiled, and need to be rebased. This is why we recommend you work off of a new branch for a PR, as described here: https://contribute.freecodecamp.org/#/how-to-setup-freecodecamp-locally?id=making-changes-locally

**Comment by yaseen221** [comment]
I was having trouble with my WSL2 that's why I used Gitpod. Will work off a new branch locally next time. Thanks @ShaunSHamilton

**Comment by ieahleen** [comment]
@yaseen221 you can still make a beanch in gitpod, choose Terminal->Open new terminal from above and make a new branch from the terminal, and work in the new branch. GitPod is still considered "locally", even if it's a cloud environment. It's because it pollutes your github fork that you shouldn't

**Comment by ShaunSHamilton** [comment]
@yaseen221 Feel free to reach out for help, if you get stuck with stuff like that on the Discord server, or on the forum. We have a few developers on these platforms who use WSL2 for freeCodeCamp deving.

File: curriculum/challenges/english/08-data-analysis-with-python/data-analysis-with-python-course/jupyter-notebooks-cells.md [hunk 1]
```diff
@@ -19,21 +19,21 @@ More resources:
 
 ## --text--
 
-Which cells are responsible for rich display?
+What are the three main types of Jupyter Notebook Cell?
 
 ## --answers--
 
-Code Cells
+Code, MarkDown, and Python
```
Comment (nhcarrigan) [inline curriculum/challenges/english/08-data-analysis-with-python/data-analysis-with-python-course/jupyter-notebooks-cells.md:26]:
```suggestion
Code, Markdown, and Python
```

File: curriculum/challenges/english/08-data-analysis-with-python/data-analysis-with-python-course/jupyter-notebooks-cells.md [hunk 2]
```diff
@@ -19,21 +19,21 @@ More resources:
...
 
 ---
 
-Markdown Cells
+Code, MarkDown, and Raw
```
Comment (nhcarrigan) [inline curriculum/challenges/english/08-data-analysis-with-python/data-analysis-with-python-course/jupyter-notebooks-cells.md:30]:
```suggestion
Code, Markdown, and Raw
```

File: curriculum/challenges/english/08-data-analysis-with-python/data-analysis-with-python-course/jupyter-notebooks-cells.md [hunk 3]
```diff
@@ -19,21 +19,21 @@ More resources:
...
 
 ---
 
-Raw Cells
+MarkDown, Python, and Raw
```
Comment (nhcarrigan) [inline curriculum/challenges/english/08-data-analysis-with-python/data-analysis-with-python-course/jupyter-notebooks-cells.md:34]:
```suggestion
Markdown, Python, and Raw
```

**Review by naomi-lgbt — CHANGES_REQUESTED** [review]
This is very much a nitpick, but can we correct the casing on `Markdown`? 🙂

**Review by naomi-lgbt — APPROVED** [review]
LGTM~! 🥳 

Congratulations on your first pull request to this project! We look forward to future contributions! Please note that we are in a code freeze at the moment, and merging will have to wait until that is lifted.

@ShaunSHamilton if you are still happy with these changes, would you mind reapproving? 🙂

**Review by ShaunSHamilton — APPROVED** [review]
LGTM 👍 

Thanks, for catching the name, @nhcarrigan . I had always thought it was _MarkDown_...

**Comment by nhcarrigan** [comment]
I wasn't sure, myself. I did some extensive searching, though, and it seems all the primary documentation refers to it as `Markdown`.

**Comment by RandellDawson** [comment]
Congrats @yaseen221 on your first pull request (PR)! 🎉

Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 📝

**Pull request merged**